<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WTC-5159473-l2" name="CXS_Support.USRC_FDM" tests="1" failures="1" timestamp="04 Mar 2019 21:43:59 GMT" time="5.662" errors="0">
  <testcase name="EndtoEndEnrollment" time="5.662" classname="CXS_Support.USRC_FDM">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &quot;enrollmentOptionsList&quot;
     but: was &quot;{
    &quot;error&quot;: &quot;invalid_request&quot;,
    &quot;error_description&quot;: &quot;validation error&quot;
}&quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "enrollmentOptionsList"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at CXS_Support.USRC_FDM.EndtoEndEnrollment(USRC_FDM.java:82)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- EndtoEndEnrollment -->
</testsuite> <!-- CXS_Support.USRC_FDM -->
